src/assets/sass/protocol/components/_button.scss

```diff
@@ -7,19 +7,34 @@
 .mzp-c-button {
     @include border-box;
     @include text-cta;
-    @include transition(background-color 100ms);
-    background-color: $color-blue-50;
-    border-radius: 3px;
-    border: 0;
+    transition: background-color 100ms, border-color 100ms, color 100ms;
+    background-color: $color-black;
+    border-radius: 2px;
+    border: 2px solid $color-black;
+    box-shadow: 0 4px 12px rgba(0, 0, 0, .1);
     color: $color-white;
+    cursor: pointer;
     display: inline-block;
     font-weight: bold;
-    padding: $padding-sm $padding-lg;
+    padding: $padding-md $padding-lg;
     text-decoration: none;
 
     &:hover,
     &:focus {
-        background-color: $color-blue-60;
+        background-color: $color-white;
+        color: $color-black;
+    }
+}
+
+.mzp-c-button.mzp-t-download {
+    background-color: $color-green-70;
+    box-shadow: none;
+    border-color: $color-green-70;
```

Comment:
Might have to set the border to none and update the padding to something like calc(#{$padding-sm} + 1px) to keep the size of this button consistent with other buttons that have border in case they are placed side by side.